server/config.js

```diff
@@ -56,6 +56,17 @@ module.exports = {
     },
     baseUrl: getRequiredEnv('PRISON_API_BASE_URL', 'https://api.nomis'),
   },
+  prisonerContactRegisteryApi: {
```

Comment:
Spelling, `registry` 👍 Needs correcting across the board.

Reply:
I agree.

Reply:
updated.